Modifications

matt-tyler opened the pull request:
- Modified wire_master_reader example to be compatible with changes to the wire library in 1.0
- Modifed FindArduino to use the correct pins_arduino.h file based on the defined board. Only added the directories to the build path, so it should remain back-compatible with older versions.

In addition :- Noted that it's printing out the board / programmer list regardless of whether it finds the path of the Arduino SDK, if the cache hasn't been built first. If it did find them, it successfully constructs the correct make file, however this is a little confusing.

Possibly update the documentation? it says to use 'make upload', but it seems like 'make {$target}-upload' is what is actually required to upload to the board.

queezythegreat commented:
Great addition, I'll try to merge in the changes when I have time to fully test out the new version 1.0.

Actually there is a upload target, if you have multiple firmware images defined in a project it will upload each target one after the other.  So when you have just one target upload == ${TARGET}-upload. The documentation is accurate when it comes to the uploading.

matt-tyler committed:
Adjustments for Arduino v1.0
- Modified wire_master_reader example to be compatible with changes to the wire library in 1.0
- Modifed FindArduino to use the correct pins_arduino.h file based on the defined board

queezythegreat commented:
Thanks for adding Arduino 1.0 support, worked like a charm with some minor modification relating to the version detection.

Sorry it took so long to integrate your changes, I was very busy last month.
